org.eclipse.lsp4e/src/org/eclipse/lsp4e/LSPEclipseUtils.java

```diff
@@ -228,13 +228,13 @@ public static CompletionParams toCompletionParams(URI fileUri, int offset, IDocu
 		if (document.getLength() > 0) {
 			try {
 				int positionCharacterOffset = offset > 0 ? offset-1 : offset;
-				String positionCharacter = document.get(positionCharacterOffset, 1);
-				if (Chars.contains(completionTriggerChars, positionCharacter.charAt(0))) {
-					param.setContext(new CompletionContext(CompletionTriggerKind.TriggerCharacter, positionCharacter));
+				String triggerCharacter = document.get(positionCharacterOffset, 1);
```

Comment:
I would leave the name of the variable as it was before. Would that be good for you?

Reply:
I'am fine with the old name as well.